solidity/contracts/interfaces/IInterchainSecurityModule.sol

```diff
@@ -7,7 +7,9 @@ interface IInterchainSecurityModule {
         ROUTING,
         AGGREGATION,
         LEGACY_MULTISIG,
-        MULTISIG
+        MULTISIG,
+        WORMHOLE,
+        NATIVE
```

Comment:
nit: I wouldn't call this "NATIVE", that very much depends on your frame of reference. Prefer "OPTIMISM"